stetho/src/main/java/com/facebook/stetho/common/Util.java

```diff
@@ -25,6 +25,12 @@
     return item;
   }
 
+  public static <T> void throwIfNull(T... items) {
```

Comment:
Let's start by adding throwIfNull() overloads for the number of parameters that we need it for, instead of using varags. In this case, I see it being called with 3 parameters. This will avoid allocation of an array for the sake of parameter validation (which needs to be as close to zero impact as possible).